### DISCUSSION TITLE
How to use wrangler with an existing project (like sveltekit) ?

### DISCUSSION THREAD

**Discussion by mkhennoussi** [cloudflare/workers-sdk#6601] [Q&A] [answered]
I didn't find in the docs how to use wrangler CLI with an existing project (like adding/generating wrangler.toml, etc). Is it possible ? 

Thanks !

**Comment by penalosa** [comment, marked as answer]
To use an existing worker with Wrangler, you can run `wrangler init --from-dash YOUR_WORKER_NAME` which will generate a `wrangler.toml` file and set up your project for local development

**Comment by slxny** [comment]
Thanks
